### PR TITLE
National ID validation, informant/mother/father DOB not after child validation

### PR DESCRIPTION
**Pull request by cibelius**
## Description

https://github.com/opencrvs/opencrvs-core/issues/8905 & https://github.com/opencrvs/opencrvs-core/issues/8917

Core PR: https://github.com/opencrvs/opencrvs-core/pull/9118

- Add National ID validation: 
  - Must be valid national ID (9 characters, only numbers)
  - Fathers national id must not match mothers national id, and vice versa
- Add informant, mother & father DOB validation, must be before childs DOB
- e2e tests are under construction [here](https://github.com/opencrvs/opencrvs-farajaland/pull/1313)
- Minor refactoring, e.g. move validators to own file
- Fix some translations

## Checklist

- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/form/v2/birth/forms/pages/mother.ts [hunk 1]
```diff
@@ -136,11 +137,20 @@ export const mother = defineFormPage({
       validation: [
         {
           message: {
-            defaultMessage: 'Must be a valid Birthdate',
+            defaultMessage: 'Must be a valid birth date',
```
Comment (makelicious) [inline src/form/v2/birth/forms/pages/mother.ts:140]: I think it was intentionally change to the format QA requires. @rikukissa 

https://preply.com/en/question/birth-date-or-birthdate

File: src/translations/client.csv [hunk 2]
```diff
@@ -2546,7 +2546,7 @@ v2.event.birth.action.declare.form.review.title,Title of the form to show in rev
 v2.event.birth.action.declare.form.section.child.field.attendantAtBirth.label,This is the label for the field,Attendant at birth,Accoucheur(se)
 v2.event.birth.action.declare.form.section.child.field.birthLocation.label,This is the label for the field,Location of birth,Lieu de naissance
 v2.event.birth.action.declare.form.section.child.field.birthType.label,This is the label for the field,Type of birth,Type d'accouchement
-v2.event.birth.action.declare.form.section.child.field.dob.error,This is the error message for invalid date,Please enter a valid date,Please enter a valid date
+v2.event.birth.action.declare.form.section.child.field.dob.error,This is the error message for invalid date,Must be a valid birth date,Doit être une date de naissance valide
```
Comment (makelicious) [inline src/translations/client.csv:2549]: Improvements. @rikukissa do we want to stick with exact same phrases

**Review by makelicious — APPROVED** [review]
Very nice, check the translations with riku

File: src/form/v2/birth/forms/pages/mother.ts (hunk 1 above)
Comment (cibelius) [reply src/form/v2/birth/forms/pages/mother.ts:140]: Screenshot 2025-04-04 at 13 08 26

The new one matches V1, so should be ok 👍 Birthdate also sounds weird, probably more of an issue on the QA requirements
